pipes/new_signal_minute.pipe

```diff
@@ -0,0 +1,24 @@
+DESCRIPTION >
+	Round Time Down to minute
+
+
+NODE signal_minute_0
+SQL >
+
+    SELECT 
+        toStartOfMinute(time) AS time,
+        network,
+        COALESCE(NULLIf(asset, ''), signal) AS signal,
+        metric,
+        value,
+        unit,
+        job_id,
+        ingested_at
+    FROM signals
+    WHERE time > '2024-10-04 10:10:00'
+
+TYPE materialized
+DATASOURCE new_signal_minute_mv
+ENGINE "MergeTree"
```

Comment:
This is added to the pipe when the view was created from the UI, let's remove all these ENGINE* settings